projectile.el

```diff
@@ -1260,8 +1260,11 @@ dirty project list."
                                 :action 'projectile-vc)))
 
 (defun projectile-files-via-ext-command (command)
-  "Get a list of relative file names in the project root by executing COMMAND."
-  (split-string (shell-command-to-string command) "\0" t))
+  "Get a list of relative file names in the project root by executing COMMAND.
+
+If `command' is nil or an empty string, return nil.
+This allows commands to be disabled."
+  (when command (split-string (shell-command-to-string command) "\0" t)))
```

Comment:
`s/when/stringp`?

Reply:
Yeah. I guess it'd be best if it was `stringp` and some non-empty check, as it doesn't make much sense to invoke an empty string command.

Reply:
Although if `shell-command-to-string` does nothing with an empty string then we are fine.

Reply:
Yeah `(shell-command-to-string "")` returns `""`; it just raises on `nil`, which was my problem.